src/stories/components/Card/styles.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Card
+
+.card
+{
+    border-radius: 5px;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
```

Comment:
Not 100% sure but maybe convert the px to rem function.
i.e. border-radius: rem(5px);

Reply:
This should stay as is. We don't want the border shadow effect to change sizes when if the font size changes